.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2.1
```

Comment:
CircleCI is having issues right now but, eventually, hopefully this will run so we can verify it: https://app.circleci.com/pipelines/github/blimmer/joist-ts-sample